confgenerator/confgenerator.go

```diff
@@ -381,14 +382,14 @@ func (l *Logging) generateFluentbitComponents(ctx context.Context, userAgent str
 		}
 	}
 	out = append(out, LoggingReceiverFilesMixin{
-		IncludePaths: []string{"${logs_dir}/logging-module.log"},
+		IncludePaths: []string{fluentbitSelfLogs(platform.FromContext(ctx))},
 		//Following: b/226668416 temporarily set storage.type to "memory"
 		//to prevent chunk corruption errors
 		BufferInMemory: true,
 	}.Components(ctx, fluentBitSelfLogTag)...)
 
 	out = append(out, LoggingReceiverFilesMixin{
-		IncludePaths:   []string{healthChecksLog(platform.FromContext(ctx))},
+		IncludePaths:   []string{path.Join("${logs_dir}", "health-checks.log")},
```

Comment:
Maybe we can also define a constant for `health-checks.log` or a function like the one for fluent-bit.